client/metro.config.js

```diff
@@ -0,0 +1,6 @@
+const { getDefaultConfig } = require('@expo/metro-config');
+
+const defaultConfig = getDefaultConfig(__dirname);
+defaultConfig.resolver.sourceExts.push('cjs');
+
+module.exports = defaultConfig;
```

Comment:
what is this used for?

Reply:
not really sure, i was following the docs: https://docs.expo.dev/guides/using-firebase/